src/ape_test/accounts.py

```diff
@@ -51,7 +51,7 @@ def _dev_accounts(self) -> List[GeneratedDevAccount]:
     @property
     def aliases(self) -> Iterator[str]:
         for index in range(self._num_of_accounts):
-            yield f"dev_{index}"
+            yield f"TEST::{index}"
```

Comment:
might make this breaking, but I am fine with that

Reply:
it never worked before anyway, so it is no really breaking. You couldn't do anything with this alias. The only place anything remote related to the alias happens is `get_user_selected_account` but that required TEST:: prefix anyway, so this just makes those the same, which somehow became apparent that they weren't in this PR

Reply:
sounds good, as long as it's thought through